docs/variants/novacustom_ns5x_adl/ec_recovery.md

```diff
@@ -0,0 +1,135 @@
+# EC firmware recovery
```

Comment:
What models this recovery procedure works for? Just NS5x?

Reply:
@pietrushnic In current state, yes. In general, all are very similar and as you can see we copy docuemnts from one model to another, chaging a few strings.

I tried to unify that but stopped at building manual. Not much point in reviewing other documents right now I think.

Reply:
Ah, sorry. Please let me know when I can help.

Reply:
Added a combined recovery doc for all models.